sched0022_to_fn() does not return a value

The function `sched0022_to_fn()` does not return a value and the compiler complains
```
  [249/267] Building C object .../src/tests/scheduler.c.obj
  .../src/tests/scheduler.c:1653:1: warning: control reaches end of non-void function [-Wreturn-type]
  }
  ^
  1 warning generated.
```

I wonder, is there just a `return sel4test_get_result();` missing?

--- a/apps/sel4test-tests/src/tests/scheduler.c
+++ b/apps/sel4test-tests/src/tests/scheduler.c
@@ -1652,6 +1652,7 @@ int sched0022_to_fn(struct env *env, helper_thread_t *thread, seL4_CPtr ep)
 
     seL4_SetMR(1, error);
     seL4_Send(ep, tag);
+    return 0;
 }
 
 /* Test that a helper thread can move itself back from another core.

